catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -269,6 +210,55 @@ const SearchForm = ({
                   Let us know what you think
                 </a>
               </p>
+              {works && (
+                <fieldset
+                  className={classNames({
+                    relative: true,
+                  })}
+                  style={{
+                    left: '1px',
+                  }}
+                >
+                  <legend
+                    className={classNames({
+                      'float-l': true,
+                      [font({ s: 'HNL4' })]: true,
+                    })}
+                    style={{ marginTop: '3px' }}
```

Comment:
One for _The Great Spacing Debate_

Reply:
What about, _The great debate of space_?
Or _The great DebAce_?